### DISCUSSION TITLE
What heppened?

### DISCUSSION THREAD

**Discussion by pes1972** [lllyasviel/stable-diffusion-webui-forge#965] [General]
After latest updates I can´t open any lora or embedding in Forge ( it worked 2 days ago). No matter what i try i get " TypeError: 'NoneType' object is not iterable" when pressing generate. Even if i don´t use any lora or embedding and just try to write "a woman" in the prompt get the same error message. Any ideas how ot fix and what´s happened?

**Comment by lllyasviel** [comment]
All tests passed for prompt grammar, break, embeddings, loras, doras etc and perfectly reproduce A1111 1.10:

Forge latest:

![image](https://github.com/user-attachments/assets/447c9dcb-44fa-4b11-a60d-85aeefdd07b0)

Automatic 1111 original 1.10

![image](https://github.com/user-attachments/assets/2e27764b-ce91-42ae-8c00-f2b33169dae8)

**Reply by pes1972** [reply]
Seems i started it while things were still updating, restarted now and it made another 3-4 update changes and now it works.

**Comment by pes1972** [comment]
.
